cli/flox/src/commands/pull.rs

```diff
@@ -93,7 +93,7 @@ impl Pull {
         match self.pull_select {
             PullSelect::New { remote } | PullSelect::NewAbbreviated { remote } => {
                 let start_message = format!(
-                    "⬇️  Remote: pulling and building {env_ref} from {host} into {into_dir}",
+                    "⬇️ Pulling {env_ref} from {host} into {into_dir}",
```

Comment:
This is the only place where we use an emoji in an spinner (rather than a final message) so I'm inclined to remove it for consistency.
```suggestion
                    "Pulling {env_ref} from {host} into {into_dir}",
```